.github/workflows/default.yml

```diff
@@ -29,111 +32,110 @@ jobs:
       - name: Docs
         run: make docs
 
+      - name: Toml
+        run: make toml
+
       - name: Porcelain
         run: make porcelain
 
-  build:
-    name: Build
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v3
+      - name: Test
+        run: make test
 
-      - uses: actions/setup-go@v4
+      - name: Lint
```

Comment:
Du hast jetzt alles in einen Job gemanscht? Damit laufen die nicht mehr parallel in der CI?

Reply:
Das war in Summe schneller als die Go Deps jedes mal zu laden. Aber mit dem Cache (Master) machen die Einzeltasks deutlich mehr Sinn und Sicht übersichtlicher. Ich hab das darauf umgestellt.